src/lib/logger-utils.js

```diff
@@ -1,7 +1,22 @@
-const { logger } = require('defra-logging-facade')
+const createDebug = require('debug')
 
 const IGNORE_PATHS = ['/public/', '/robots.txt']
 
+// if DEBUG is not set, default to show error and info
+if (!process.env.DEBUG) {
+  createDebug.enable('rcr-frontend:error,rcr-frontend:info')
+}
+createDebug.inspectOpts.colors = true
+
+const info = createDebug('rcr-frontend:info')
+info.color = 2 // green
```

Comment:
I'm surprised Sonar hasn't picked up your use of magic numbers, but it'd be preferable to declare these as constants, or, even better, as an enum at the top of the file and get rid of the comments